prime-router/src/main/kotlin/cli/TestReportStream.kt

```diff
@@ -34,9 +34,15 @@ import java.net.HttpURLConnection
 import java.nio.file.Files
```

Comment:
To have each test return a pass/fail summary, I ended up doing a giant refactor of this whole class.  Each test is now its own Class, and is a child of an abstract class called CoolTest.  Everything moved around.